### PR TITLE
Film page

### PR DESCRIPTION
**Pull request by richerstars**

### CODE REVIEW

File: src/components/MainLayout/MainLayout.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ class MainLayout extends Component {
     constructor(props) {
         super(props);
     }
+
```
Comment (nataliazhuravlova) [inline src/components/MainLayout/MainLayout.js]: empty

File: src/components/MoviePage/GenreDetails/GenreDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const GenreDetails = ({genres}) => {
```
Comment (nataliazhuravlova) [inline src/components/MoviePage/GenreDetails/GenreDetails.jsx]: without return

File: src/components/MoviePage/MovieDetails/MovieDetails.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import '../MoviePage.scss';
+
+const MovieDetails = ({film,hours,minutes}) => {
```
Comment (nataliazhuravlova) [inline src/components/MoviePage/MovieDetails/MovieDetails.jsx]: without return

File: src/components/MoviePage/MovieDetails/MovieDetails.jsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    return (
+        <ul className="base-gaps" id="info">
+            <li className="small-info info-block-size">adult: {film.adult
+                ? 'true'
+                : 'false'
```
Comment (nataliazhuravlova) [inline src/components/MoviePage/MovieDetails/MovieDetails.jsx]: what is that????

File: src/components/MoviePage/MoviePage.jsx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    componentDidMount() {
+        this.getMovieFromApi();
+    }
+
+    getMovieFromApi = async () => {
+        try {
+            const res = await axios.get(`${MOVIE_BY_ID}${this.props.match.params.id}`);
+            this.setState({film: res.data});
+            this.formatRuntime();
+        } catch (error) {
```
Comment (nataliazhuravlova) [inline src/components/MoviePage/MoviePage.jsx]: unnessasary catch

File: src/components/MoviePage/MoviePage.jsx [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+        const minutes = this.state.film.runtime % 60;
+        this.setState({hours: hours, minutes: minutes});
+    }
+
+
+    render() {
+        const styles = {
+            background: `linear-gradient( rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7) ), url(${URL_IMG + this.state.film.backdrop_path})`,
+            backgroundSize: 'cover'
+        };
```
Comment (nataliazhuravlova) [inline src/components/MoviePage/MoviePage.jsx:50]: in sass

File: src/components/MoviePage/OverviewDetails/OverviewDetails.jsx [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const OverviewDetails = ({overview}) => {
```
Comment (nataliazhuravlova) [inline src/components/MoviePage/OverviewDetails/OverviewDetails.jsx]: return

File: src/components/Movies/Movies.js [hunk 8]
```diff
@@ -46,9 +46,9 @@ class Movies extends Component {
                     {this.state.isLoading ? (
                         <Loader />
                     ) : this.state.movies.length ? 
-                        (this.state.movies.map((element) => (
+                        (this.state.movies.map((element,index) => (
                             <MovieCard
-                                key={element.id}
+                                key={index}
```
Comment (nataliazhuravlova) [inline src/components/Movies/Movies.js]: why index

File: src/components/common/Loader/Loader.jsx [hunk 9]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const Loader = () => {
```
Comment (nataliazhuravlova) [inline src/components/common/Loader/Loader.jsx:3]: use created one

File: src/components/MainLayout/MainLayout.js (hunk 1 above)
Comment (richerstars) [reply src/components/MainLayout/MainLayout.js]: done

File: src/components/MoviePage/GenreDetails/GenreDetails.jsx (hunk 2 above)
Comment (richerstars) [reply src/components/MoviePage/GenreDetails/GenreDetails.jsx]: done

File: src/components/MoviePage/MovieDetails/MovieDetails.jsx (hunk 3 above)
Comment (richerstars) [reply src/components/MoviePage/MovieDetails/MovieDetails.jsx]: done

File: src/components/MoviePage/MovieDetails/MovieDetails.jsx (hunk 4 above)
Comment (richerstars) [reply src/components/MoviePage/MovieDetails/MovieDetails.jsx]: done

File: src/components/MoviePage/OverviewDetails/OverviewDetails.jsx (hunk 7 above)
Comment (richerstars) [reply src/components/MoviePage/OverviewDetails/OverviewDetails.jsx]: done

File: src/components/Movies/Movies.js (hunk 8 above)
Comment (richerstars) [reply src/components/Movies/Movies.js]: done

File: src/components/common/Loader/Loader.jsx (hunk 9 above)
Comment (richerstars) [reply src/components/common/Loader/Loader.jsx:3]: done

**Pull request merged**